store/sharding/helpers.go

```diff
@@ -0,0 +1,32 @@
+package sharding
+
+import "github.com/forta-network/forta-node/config"
+
+const (
+	keyDefaultChainSetting = "default"
+	defaultShardID         = 0
+	minShardCount          = 1
+	defaultTargetCount     = 3
```

Comment:
FYI - default target is 6 https://github.com/forta-network/forta-assigner/blob/9d2091deb6ab51ec0d1657e6607206854afadc39/domain/chain.go#L8

Though I think I want the assigner to switch to 3, but still, it's 6 for now

Reply:
If we deploy a v2 bot in dev right now, we get 3 assignments, not 6?

Reply:
We need to be crystal clear that the default is always 3 for every scanned source - can't afford to default to 6 for an unknown reason. Chain ID and shard ID detection is strictly based on how many assignments are expected.